src/NUnitTestAdapter/TestConverter.cs

```diff
@@ -116,8 +116,15 @@ public IList<VSTestResult> GetVSTestResults(XmlNode resultNode)
                     results.Add(result);
             }
 
-            return results;
+            return new TestResultSet {TestCaseResult = testcaseResult, TestResults = results};
+        }
+
+        public struct TestResultSet
+        {
+            public IList<VSTestResult> TestResults { get; set; }
+            public TestResult TestCaseResult { get; set; }
```

Comment:
This all gets a bit confusing without a prefix to distinguish NUnit from VS results.

Reply:
Good point, I'll fix.